xml/deploy_cephadm.xml

```diff
@@ -773,14 +773,14 @@ o- time_server ................................................ [enabled]
        xlink:href="https://documentation.suse.com/suse-caasp/4.5/single-html/caasp-airgap/"/>.
      </para>
     </tip>
-     <para>
-      Cloud Native Application Development and Delivery methods require a
-      registry and a CI/CD (Continuous Integration/Delivery) instance for the
-      development and production of container images. You can use a private
-      registry in this instance. For more information, see
-      <link
+    <para>
+     Cloud Native Application Development and Delivery methods require a
+     registry and a CI/CD (Continuous Integration/Delivery) instance for the
```

Comment:
I think it is a little odd that the definition of the initialism does not match. I know what it means and I think most readers will so this is minor, but it looks strange to me.